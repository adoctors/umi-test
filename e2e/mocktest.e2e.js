const BASE_URL = `http://localhost:${process.env.PORT || 9001}/mock/test`;

describe('mock-test', () => {
  beforeAll(async () => {
    jest.setTimeout(1000000);
  });
  it('it should mock-test', async () => {
    await page.goto(BASE_URL);
    await page.waitForSelector('p', {
      timeout: 5000,
    });
    const text = await page.evaluate(() => document.getElementsByTagName('p')[0].innerText);
    await page.screenshot({ path: 'mock-test.png' });
    expect(text).toContain('mock');
  });
});