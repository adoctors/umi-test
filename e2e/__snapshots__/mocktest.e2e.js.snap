// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mock-test it should mock-test 1`] = `"mock"`;
